README.md

```diff
@@ -6,10 +6,68 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for managing and monitoring components
+within a microgrid. It offers various functionalities related to microgrid
+management, including metadata retrieval, component listing, setting power
+levels, and controlling the state of components. The API is structured around
+the concept of a microgrid, which is a localized energy system composed of
+various interconnected components.
 
-This repository provides the core gRPC and protobuf spec together with
-support for generating dedicated API packages for various languages.
+
+## Objective
+
+The objective of this API is to provide a standardized interface for
+interacting with and controlling components within a microgrid. It aims to
```

Comment:
I think whats missing is that its vendor agnostic meaning it provides an abstract interface independing of the the specific hardware vendor. Thats the biggest USP this provides.

Reply:
Done.